tck/src/main/java/org/eclipse/microprofile/fault/tolerance/tck/CircuitBreakerTest.java

```diff
@@ -44,244 +46,469 @@
 public class CircuitBreakerTest extends Arquillian {
 
     private @Inject CircuitBreakerClientWithDelay clientForCBWithDelay;
+    private @Inject CircuitBreakerClassLevelClientWithDelay clientForClassLevelCBWithDelay;
     private @Inject CircuitBreakerClientNoDelay clientForCBNoDelay;
     private @Inject CircuitBreakerClientDefaultSuccessThreshold clientForCBDefaultSuccess;
     private @Inject CircuitBreakerClientHigherSuccessThreshold clientForCBHighSuccess;
     
     @Deployment
     public static WebArchive deploy() {
-        JavaArchive testJar = ShrinkWrap
-                .create(JavaArchive.class, "ftCircuitBreaker.jar")
-                .addClasses(CircuitBreakerClientWithDelay.class,CircuitBreakerClientNoDelay.class,
-                                CircuitBreakerClientDefaultSuccessThreshold.class,CircuitBreakerClientHigherSuccessThreshold.class)
-                .addAsManifestResource(EmptyAsset.INSTANCE, "beans.xml")
-                .as(JavaArchive.class);
-
-        WebArchive war = ShrinkWrap
-                .create(WebArchive.class, "ftCircuitBreaker.war")
-                .addAsLibrary(testJar);
+        JavaArchive testJar = ShrinkWrap.create(JavaArchive.class, "ftCircuitBreaker.jar")
```

Comment:
Its very hard to read what the actual diff is in this commit, since there's a lot of reformatting.  I would suggest creating a separate commit that deals with the reformatting.